packages/eas-cli/src/commands/build/run.ts

```diff
@@ -167,19 +167,23 @@ async function maybeGetBuildAsync(
     !flags.runArchiveFlags.url &&
     !flags.runArchiveFlags.latest
   ) {
-    return await listAndSelectBuildsOnAppAsync(graphqlClient, flags.selectedPlatform, {
-      projectId,
-      projectDisplayName: await getDisplayNameForProjectIdAsync(graphqlClient, projectId),
-      filter: {
-        platform: flags.selectedPlatform,
-        distribution: distributionType,
-        status: BuildStatus.Finished,
-      },
-      queryOptions: paginatedQueryOptions,
-      selectPromptDisabledFunction: build => !isRunnableOnSimulatorOrEmulator(build),
-      warningMessage:
-        'Artifacts for this build have expired and are no longer available, or this is not a simulator/emulator build.',
-    });
+    return (
+      (await listAndSelectBuildOnAppAsync(graphqlClient, {
+        projectId,
+        title: `Select ${flags.selectedPlatform === AppPlatform.Ios ? 'iOS' : 'Android'} ${
+          flags.selectedPlatform === AppPlatform.Ios ? 'simulator' : 'emulator'
+        } build to run for ${await getDisplayNameForProjectIdAsync(graphqlClient, projectId)} app`,
+        filter: {
+          platform: flags.selectedPlatform,
+          distribution: distributionType,
+          status: BuildStatus.Finished,
+        },
+        paginatedQueryOptions,
+        selectPromptDisabledFunction: build => !isRunnableOnSimulatorOrEmulator(build),
+        selectPromptWarningMessage:
+          'Artifacts for this build have expired and are no longer available, or this is not a simulator/emulator build.',
+      })) ?? null
```

Comment:
IMO this is not readable. The function call is too long. I'd assign the result to a variable and then write `return xxx ?? null;`